<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ФИО исполнителя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="executorLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Проект:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="projectLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Задание:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="taskLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Дата начала:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startDateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата окончания:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endDateEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="add_task_button">
        <property name="text">
         <string>Добавить задачу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_task_button">
        <property name="text">
         <string>Удалить задачу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_task_button">
        <property name="text">
         <string>Редактировать задачу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sort_tasks_button">
        <property name="text">
         <string>Сортировать по дате</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="load_tasks_button">
        <property name="text">
         <string>Загрузить из файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_tasks_button">
        <property name="text">
         <string>Сохранить в файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getTasksByExecutorButton">
        <property name="text">
         <string>Задачи за сегодня</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="get_total_work_time_button">
        <property name="text">
         <string>Суммарное время</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="taskTableWidget">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО исполнителя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Проект</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Задание</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата начала</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата окончания</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
